main/utils/devices.js

```diff
@@ -30,23 +30,34 @@ const getAudioDevices = async () => {
       return 0;
     }).map(device => ({id: device.uid, name: device.name}));
   } catch (error) {
-    const devices = await aperture.audioDevices();
+    try {
+      const devices = await aperture.audioDevices();
+
+      if (!Array.isArray(devices)) {
+        const Sentry = require('./sentry');
+        Sentry.captureException(new Error(`devices is not an array: ${JSON.stringify(devices)}`));
+        showError(error);
+        return [];
+      }
 
-    if (!Array.isArray(devices)) {
-      const Sentry = require('./sentry');
-      Sentry.captureException(new Error(`devices is not an array: ${JSON.stringify(devices)}`));
+      return devices;
```

Comment:
This is what caused the crash, the rest is just additional checks and fallbacks for devices